src/main/java/com/novi/backend_spring_boot_tech_it_easy_controller/exceptions/ExceptionController.java

```diff
@@ -0,0 +1,22 @@
+package com.novi.backend_spring_boot_tech_it_easy_controller.exceptions;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+@ControllerAdvice
+public class ExceptionController {
+
+    @ExceptionHandler(TelevisionNotFoundException.class)
```

Comment:
Ik zie wel een RecordNotFoundException in je project, maar geen TelevisionNotFoundException. Het wordt ook nooit opgegooid in je controller.

Een exception gooien doe je met het `throw` keyword, bijvoorbeeld 
`throw new RecordNotFoundException("beschrijf wat er fout is gegaan en wat de gebruiker anders kan doen")`. 
Dit gebruik je op plekken waar je applicatie niet verder kan werken, bijvoorbeeld als je gebruiker een ID opvraagt die niet bestaat.